### ISSUE TITLE
Hi, i am getting error 401 Authorization Required nginx/1.14.1, whenever i try to preview my sites, when i use python3 -m html.server, it says No module named html.server, help?!

### ISSUE THREAD

**Issue by Valencia-08** [gitpod-io/gitpod#2261]

**Comment by Valencia-08** [comment]
Hi, i am getting error 401 Authorization Required
nginx/1.14.1, whenever i try to preview my sites, when i use python3 -m html.server, it says No module named html.server, help?!

**Comment by Valencia-08** [comment]
*http.server https://8000-ec6b8096-b679-469f-9405-2e61495858af.ws-eu01.gitpod.io/ thats what comes up

**Comment by jankeromnes** [comment]
Hi @Valencia-08! Thank you for reaching out.

So, I suppose that you indeed mean `python3 -m http.server 8000` (and not `html.server`, which will explain the "No module named 'html'" error).

Unfortunately, I'm not able to see the error page with the link you shared, because workspace previews are both private (you'll need to explicitly share your workspace, or make the port public, for others to be able to use that URL) and temporary (so even if you do share the workspace or port, and post the URL, chances are by the time someone tries the URL your workspace will have already stopped).

Long story short, could you please post a screenshot of your error? Or even better, a link to your repository, or a Gitpod Snapshot?

**Comment by Valencia-08** [comment]
hi thanks for getting back to me, I think I have resolved the issue but if it presumes I will screenshot it and attach the images here.

**Comment by jankeromnes** [comment]
@Valencia-08 Good news! Thanks for the update, and yes please do post screenshots & repository URL if this problem re-occurs.

I'll close this issue for now, but will re-open it if the problem comes back.
